packages/core/src/actions/cancel.ts

```diff
@@ -33,9 +33,7 @@ function resolveCancel(
 }
 
 function executeCancel(actorScope: AnyActorScope, resolvedSendId: string) {
-  actorScope.defer(() => {
-    actorScope.system.scheduler.cancel(actorScope.self, resolvedSendId);
-  });
+  actorScope.system.scheduler.cancel(actorScope.self, resolvedSendId);
```

Comment:
do we have some new test that shows this is a desired change?

Reply:
There is no reason to defer it - it's a clean-up